doc/Makefile

```diff
@@ -0,0 +1,194 @@
+# Makefile for Sphinx documentation
```

Comment:
we seem to have two Makefiles now.  We probably don't need this one (also hte make.bat in here) but look through it as if you were merging this Makefile and the one in source from cookiecutter, and raise any aquestions that come up.

Reply:
Generally I've been basing the structure off diffpy.pdfmorph and diffpy.utils. They both have `Makefile` directly under `doc/`. Also, diffpy.pdfmorph still has the make.bat file so wasn't sure if it was still needed here.